corehq/apps/accounting/tests/test_models.py

```diff
@@ -123,13 +132,17 @@ def test_is_hidden_to_ops(self):
 
     def test_next_subscription(self):
         this_subscription_date_end = self.subscription.date_end
-        already_canceled_future_subscription, _ = generator.generate_domain_subscription_from_date(
-            this_subscription_date_end, self.account, self.domain.name,
-            subscription_length=0
+        already_canceled_future_subscription= generator.generate_domain_subscription(
```

Comment:
F841 local variable 'already_canceled_future_subscription' is assigned to but never used